packages/Radio/src/components/Group/Group.js

```diff
@@ -30,21 +27,33 @@ const defaultProps = {
   a11yText: "",
   canDeselect: false,
   children: null,
-  defaultCheck: () => {},
   isDisabled: false,
   size: ShirtSizes.MEDIUM,
 };
 
 function Group(props) {
-  const { a11yText, canDeselect, children, defaultCheck, onChange, ...moreGroupProps } = props;
-  const defaultCheckedId = React.Children.toArray(children).find(defaultCheck).props.value.id;
-  const [checkedId, setCheckedId] = React.useState(defaultCheckedId || null);
-  const deselectableId = id => (checkedId === id ? null : id);
+  const { a11yText, canDeselect, children, onChange, ...moreGroupProps } = props;
+  const defaultCheckedIndex = React.Children.toArray(children).findIndex(child => child.props.defaultIsSelected);
+  const selectedIndex = React.Children.toArray(children).findIndex(child => child.props.isSelected);
+
+  const defaultIndex = () => {
```

Comment:
Why is this a function?

Reply:
Previously I was evaluating other props as well, changed to just use the variable now